module/Company/src/Company/Controller/AdminController.php

```diff
@@ -193,6 +194,78 @@ public function addJobAction()
     }
 
     /**
+     * Action that displays a form for editing a category
+     *
+     *
+     */
+    public function editCategoryAction()
+    {
+        // Get useful stuff
+        $companyService = $this->getCompanyService();
+        $categoryForm = $companyService->getCategoryForm();
+
+        // Get parameter
+        $categoryId = $this->params('categoryID');
+
+        // Get the specified company
+        $categories = $companyService->getAllCategoriesById($categoryId);
+
+        // If the company is not found, throw 404
+        if (empty($categories)) {
+            $company = null;
+            return $this->notFoundAction();
+        }
+
+        // Initialize form
+        $categoriesDict = [];
+        foreach ($categories as $category) {
+            $categoriesDict[$category->getLanguage()] = $category;
+        }
+        $categoryForm->bind($categoriesDict);
+        $categoryForm->setAttribute(
+            'action',
+            $this->url()->fromRoute(
+                'admin_company/editCategory',
+                [
+                    'categoryID' => $categoryId,
+                ]
+            )
+        );
+        // Handle incoming form data
+        $request = $this->getRequest();
+        if ($request->isPost()) {
+            if ($companyService->saveCategory(
+                $request->getPost()
+            )){
+                return $this->redirect()->toRoute(
```

Comment:
Perhaps it would be better to show some sort of confirmation here. Now it is not clear that the category is saved since the edit page looks exactly the same as the create page.

Reply:
The h1 on the page changes.

However, the same thing happens at other places in the company module too, so I guess it would be a good idea to open a separate issue to fix this in other places too.